ros/lunar/debian/stretch/ros-core/Dockerfile

```diff
@@ -14,24 +14,17 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 4B63CF8FDE
 # setup sources.list
 RUN echo "deb http://snapshots.ros.org/lunar/final/debian stretch main" > /etc/apt/sources.list.d/ros1-snapshots.list
 
-# install bootstrap tools
-RUN apt-get update && apt-get install --no-install-recommends -y \
-    python-rosdep \
-    python-rosinstall \
-    python-vcstools \
-    && rm -rf /var/lib/apt/lists/*
-
 # setup environment
 ENV LANG C.UTF-8
 ENV LC_ALL C.UTF-8
 
-# bootstrap rosdep
-RUN rosdep init \
-    && rosdep update --include-eol-distros
+ENV ROS_DISTRO lunar
```

Comment:
Not sure if we should change EOL Dockerfile. I not against it, as I like consistency, but we should check for side effects.